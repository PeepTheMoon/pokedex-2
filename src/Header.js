import React, { Component } from 'react'

export default class Header extends Component {
    render() {
        return (
            <div className="header">
                <h1 className="header-h1">
                    Welcome to the Pokedex!
                </h1>
                <h2 className="header-h2">
                    Search for Pokemon based on their name, type, attack, and defense.
                </h2>
            </div>
        )
    }
}
