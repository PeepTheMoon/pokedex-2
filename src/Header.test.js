import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header.js';


it('renders single pokemon item details', () => {
    const wrapper = shallow(<Header/>);
  
    expect(wrapper).toMatchSnapshot();
  });