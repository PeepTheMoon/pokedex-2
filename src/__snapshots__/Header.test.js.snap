// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders single pokemon item details 1`] = `
<div
  className="header"
>
  <h1
    className="header-h1"
  >
    Welcome to the Pokedex!
  </h1>
  <h2
    className="header-h2"
  >
    Search for Pokemon based on their name, type, attack, and defense.
  </h2>
</div>
`;
